tests/test_encoding.py

```diff
@@ -287,7 +287,7 @@ def test_encode_overlapping(self) -> None:
             parameters=NamedItemList([param1, param2, param3]),
             byte_size=None,
         )
-        self.assertEqual(req.encode(), bytearray([0x12, 0x34, 0x56]))
+        self.assertEqual(req.encode(), bytearray([0x33, 0x75, 0x56]))
```

Comment:
why?

Reply:
overlapping parameters result in undefined behaviour and I was recently persuated that it is better not to raise an exception in this case, not even in strict mode... ;)